### ISSUE TITLE
DeprecationWarning

### ISSUE THREAD

**Issue by rakeshchoyal92** [nficano/humps#110]
I'm getting the following warning with python 3.8.2

 ```
DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated since Python 3.3, and in 3.9 it will stop working
    from collections import Mapping
```

**Comment by moritzmeister** [comment]
This got fixed by #79, so should be in the next release.
Officially, humps doesn't support python 3.8 yet as far as I see.

**Comment by nphilou** [comment]
Hello, thank you for opening an issue, it was indeed fixed, I will try to do a release on github, a pypi release will take more time. 

Concerning python 3.8, I tried it on this branch https://github.com/nficano/humps/tree/python38, should be in the next release also!
